core/arch/arm/plat-imx/a9_plat_init.S

```diff
@@ -100,8 +100,8 @@ UNWIND(	.fnstart)
 	movt r0, #0x0002
 	write_nsacr r0
 
-	movw r0, #0x0000
-	movt r0, #0x0001
+	movw r0, #0x0001
```

Comment:
Well, that doesn't match the code style of the rest of the file. Shall I change it entirely?